l10n-pocket/en/pocket/platforms.ftl

```diff
@@ -0,0 +1,36 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### iOS URL: https://www-dev-pocket.allizom.org/ios/
+### iOS URL: https://www-dev-pocket.allizom.org/android/
+### iOS URL: https://www-dev-pocket.allizom.org/safari/
+### iOS URL: https://www-dev-pocket.allizom.org/chrome/
+### iOS URL: https://www-dev-pocket.allizom.org/opera/
+### iOS URL: https://www-dev-pocket.allizom.org/edge/
+### iOS URL: https://www-dev-pocket.allizom.org/welcome/
+
+
+pocket-platform-get-app = Get the app
+pocket-platform-install = Install
+pocket-platform-log-in = Log in
+
+pocket-platform-ios-title = { -brand-name-pocket } for { -brand-name-ios }
+pocket-platform-ios-view-list = View your list on your iPhone or iPad. Download the free { -brand-name-pocket } app for { -brand-name-ios } from the { -brand-name-apple-app-store }.
+
+pocket-platform-android-title = { -brand-name-pocket } for { -brand-name-android }
+pocket-platform-android-view-list = View your list on your { -brand-name-android} device. Download the free Pocket app from { -brand-name-google-play }.
+
+
+pocket-platform-chrome-title = { -brand-name-pocket } for { -brand-name-chrome}
+
+pocket-platform-safari-title = { -brand-name-pocket } for { -brand-name-safari}
+
+pocket-platform-opera-title = { -brand-name-pocket } for { -brand-name-opera }
+
+pocket-platform-edge-title = { -brand-name-pocket } for { -brand-name-edge }
+
+pocket-platform-browser-installing = Installing the Pocket browser extension installs buttons that let you save items with one click.
```

Comment:
All `Pocket` in these strings should be replaced by the term

Reply:
Ah, good catch